examples/cslc_s1_sample_runconfig-v2.0.0-er.2.0.yaml

```diff
@@ -0,0 +1,174 @@
+        # should align with the similar sections of the SAS RunConfig
+        PGE:
+            PGENameGroup:
+                # Name of the PGE for use with this RunConfig, should always be
+                # CSLC_S1_PGE when using with the CSLC-S1 PGE
+                PGEName: CSLC_S1_PGE
+
+            InputFilesGroup:
+                # List of input files
+                # Must be a list containing the path to the input SAFE zip file(s),
+                # as well as the path to the Orbit Ephemerides file(s)
+                # Paths must correspond to the file system within the Docker container
+                InputFilePaths:
+                    - /home/compass_user/input_data/S1A_IW_SLC__1SDV_20220501T015035_20220501T015102_043011_0522A4_42CC.zip
+                    - /home/compass_user/input_data/S1A_OPER_AUX_POEORB_OPOD_20220521T081912_V20220430T225942_20220502T005942.EOF
+
+            DynamicAncillaryFilesGroup:
+                # Map of ancillary file types to paths to the file
+                # Paths must correspond to the file system within the Docker container
+                AncillaryFileMap:
+                    # Path to the Digital Elevation Model
+                    # Must be either a single .tif, or a .vrt that points to one
+                    # or more .tif files
+                    dem_file: /home/compass_user/input_data/dem_4326.tiff
+
+            ProductPathGroup:
+                # Path to where output products should be stored
+                # Must correspond to the file system within the Docker container,
+                # and must have write permissions for the User/Group used with
+                # the "Docker run" command
+                OutputProductPath: /home/compass_user/output_dir
+
+                # Path to a scratch directory for the PGE and SAS to store
+                # intermediate files that will not be needed after PGE execution
+                # completes
+                # Must correspond to the file system within the Docker container,
+                # and must have write permissions for the User/Group used with
+                # the "Docker run" command
+                ScratchPath: /home/compass_user/output_dir/scratch_dir
+
+            PrimaryExecutable:
+                # Identifier for the PGE executable, should always be CSLC_S1 for
+                # this PGE
+                ProductIdentifier: CSLC_S1
+
+                # Path to the executable to run, path must be reachable from
+                # within the Docker container (i.e. findable with a "which" command)
+                ProgramPath: conda
+
+                # List of command-line options to use with ProgramPath
+                ProgramOptions:
+                    - run
+                    - -n
+                    - COMPASS
+                    - s1_cslc.py
+
+                # The Error Code base value added to the offset values to make
```

Comment:
The unique-error-code-per-PGE convention was something inherited from NISAR for our initial PGE design. I'm guessing NISAR wanted it since a single PGE can have multiple "workflows", but for OPERA it might not be necessary in the long run. As far as I know, no one is interpreting the codes as of yet.

Reply:
Thanks Scott. Something to keep in mind as we add more PGE's - how can we either set up a simple lookup table that accounts for versioned codes or simplify the codes themselves.